src/org/jitsi/impl/neomedia/RTPConnectorOutputStream.java

```diff
@@ -645,6 +675,7 @@ private boolean write(RawPacket[] pkts)
             {
                 if (success)
                 {
+                    rateStatistics.update(pkt.getLength(), now);
                     if (!send(pkt))
```

Comment:
I didn't get why rateStatistics.update() is before send(). I trust you know why you have it that way so I went and committed it.